### PR TITLE
Delete blank comment line after coding cookie

### PR DESCRIPTION
**Pull request by verhovsky**
https://github.com/asottile/pyupgrade/issues/219

### CODE REVIEW

File: pyupgrade.py [hunk 1]
```diff
@@ -745,6 +746,9 @@ def _fix_tokens(contents_text, min_version):
             del tokens[i]
             if tokens[i].name == 'NL':  # pragma: no branch (old PY2)
                 del tokens[i]
+                # pragma: no branch (old PY2)
```
Comment (verhovsky) [inline pyupgrade.py]: No clue what this line does or if it's necessary.

File: pyupgrade.py [hunk 2]
```diff
@@ -93,8 +93,9 @@ def _convert_tup(tup):  # type: (DotFormatPart) -> str
     return j.join(_convert_tup(tup) for tup in parsed)
 
 
-# if py3+ use tokenize.cookie_re
+# if py3+ use tokenize.cookie_re and tokenize.blank_re
 cookie_re = re.compile('^[ \t\f]*#.*?coding[:=][ \t]*([-_.a-zA-Z0-9]+)')
+blank_re = re.compile(r'^[ \t\f]*(?:[#\r\n]|$)', re.ASCII)
```
Comment (verhovsky) [inline pyupgrade.py]: I don't know if the `re.ASCII` part makes sense, I just copy pasted it from `tokenize`.

File: pyupgrade.py [hunk 3]
```diff
@@ -745,6 +746,9 @@ def _fix_tokens(contents_text, min_version):
             del tokens[i]
             if tokens[i].name == 'NL':  # pragma: no branch (old PY2)
                 del tokens[i]
+                # pragma: no branch (old PY2)
+                if blank_re.match(tokens[i].src):
+                    del tokens[i]
```
Comment (verhovsky) [inline pyupgrade.py]: I think technically there should be another nested check for `if tokens[i].name == 'NL':` here, but I guess the leading empty line trimming takes care of it? Seems to work anyway.

**Comment by asottile** [comment]
via https://github.com/asottile/pyupgrade/issues/219#issuecomment-548774459

**Pull request closed without merging**